### ISSUE TITLE
Use default admin form?

### ISSUE THREAD

**Issue by johanovic** [Dunedan/django-lockdown#16]
Is it possible to use the default admin login form in stead of a custom form?

When I use login_required without django-lockdown the redirect is to the login form that is used to access the admin pages. Is it possible to replicate that behaviour with lockdown?

**Comment by Dunedan** [comment]
I'm not entirely sure I understand your use case, but have you looked into the provided [`lockdown.forms.AuthForm`](https://github.com/Dunedan/django-lockdown/blob/83cab3fb035d0b79085d82a24c7159e9b883fade/lockdown/forms.py#L50)? This offers a form for authenticating against your existing users (also see https://github.com/Dunedan/django-lockdown#lockdown_form).

If that's not what you intend, it'd be good if you could elaborate your use case a bit more.

**Comment by johanovic** [comment]
Use case is a site that only has a single public page, but thousands of internal pages. All these are protected with LoginRequired etc., but sometimes a single page isn't (which is a bug). As an extra security measure I want to use django-lockdown to lock down *all* of the pages. Now I want the login page to be the standard one I'm using which is the one for the admin.

I don't want to duplicate the admin form and design, I just want to redirect to that form. So I have looked at using the django.contrib.auth.forms.AuthenticationForm as LOCKDOWN_FORM which works, but it is a page without design. I don't want to maintain two designs of the login form and hence I'd prefer for django-lockdown to redirect to the normal django admin form. 

I'm starting to hesitate, but I might actually have had this working in the past, but I can't recall.

**Comment by johanovic** [comment]
Essentially I want to redirect to settings.LOGIN_URL if the site is locked.

**Comment by johanovic** [comment]
Any ideas on the matter? I tried adding /login/ to the LOCKDOWN_URL_EXCEPTIONS, but that then only works for /login/. Ideally there should be a setting to determine what LOGIN_URL django-lockdown should redirect to. Or am I missing something?

**Comment by Dunedan** [comment]
While I'm not aware of an option to do a HTTP-based redirect, I can think of at least two options right now, which could work for you:

1. Use a custom form template and just include the one, you're already using for your login page. I just tried it out and it works fine. To achieve that I had to place a `lockdown/form.html` with the following content in the global template directory:

        {% include "admin/login.html" %}

   With this solution you don't have a redirect, but your form looks nice and shiny without the need to keep a duplicate of your existing login form template around.

2. Use a HTML redirect. This again requires supplying a custom template for the form, but this time you'd do a [Meta refresh](https://en.wikipedia.org/wiki/Meta_refresh) instead. While that'd be a redirect, the downside would be that users wouldn't be redirected to the page they were originally trying to visit (although that could be achievable by using Javascript to dynamically add a `next` parameter to the URL to redirect to).
